app/__init__.py

```diff
@@ -48,6 +48,21 @@ def create_app(config_name):
 
     csrf.init_app(application)
 
+    @csrf.error_handler
+    def csrf_handler(reason):
+        if 'user_id' not in session:
+            application.logger.info(
+                u'csrf.session_expired: Redirecting user to log in page'
+            )
+
+            return application.login_manager.unauthorized()
+
+        application.logger.info(
+            u'csrf.invalid_token: Aborting request, user_id: %s', session['user_id']
+        )
+
+        abort(400, reason)
```

Comment:
That's what the original error handler did. 400-403 are covered by the same metric right now, so I think it's ok. We can always use the log message for additional alerts.